XamarinCommunityToolkit/Converters/ListIsNotNullOrEmptyConverter.shared.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections;
+using System.Globalization;
+using Xamarin.Forms;
+using System.Linq;
+
+namespace XamarinCommunityToolkit.Converters
+{
+    /// <summary>
+    /// Converts the incoming value to a boolean indicating whether or not the value is not null and not empty.
+    /// </summary>
+    public class ListIsNotNullOrEmptyConverter : IValueConverter
+    {
+        /// <summary>
+        /// Converts the incoming value to a boolean indicating whether or not the value is not null and not empty.
+        /// </summary>
+        /// <param name="value">The value to convert.</param>
+        /// <param name="targetType">The type of the binding target property.</param>
+        /// <param name="parameter">Additional parameter for the converter to handle. Not implemented.</param>
+        /// <param name="culture">The culture to use in the converter.</param>
+        /// <returns>A boolean indicating if the incoming value is not null and not empty.</returns>
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture) => value != null && value is IEnumerable list && list.GetEnumerator().MoveNext();
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture) => throw new NotImplementedException();
```

Comment:
We should reach an agreement regarding the code style. I mean perhaps it makes sense to write **=>** from the new line.

```csharp
public object Method(object parameter)
    => new object();
```

Reply:
Agreed. Will put that in for each.